rswag-specs/lib/rswag/specs/swagger_formatter.rb

```diff
@@ -34,6 +36,7 @@ def example_group_finished(notification)
         return unless metadata.key?(:response)
 
         swagger_doc = @config.get_swagger_doc(metadata[:swagger_doc])
```

Comment:
I'm not sure how this will behave if you don't define a `swagger_doc` in your test, but we didn't have any in our codebase so I wasn't able to test it out under this circumstance.

Also, if you have suggestions for tests we could write for this change, let me know!

Reply:
Dang, it appears the tests caught something in my code but I'm at a loss as to why it doesn't work.  If a maintainer has a suggestion I'd be happy to investigate.